Makefile

```diff
@@ -199,7 +199,7 @@ update/copyright:
 .PHONY: gen/code
 gen/code:
 	# NOTE: Specify a specific version temporally until the next release.
-	docker run --rm -v ${PWD}:/repo -it --entrypoint ./tool/codegen/codegen.sh ghcr.io/pipe-cd/codegen@sha256:fc3db505ef8dbf287b90aafed8c28246d2cca06bda2b43893a3059719fe9fff4 /repo #v0.44.0-38-g7229285
+	docker run --rm -v ${PWD}:/repo -it --entrypoint ./tool/codegen/codegen.sh ghcr.io/pipe-cd/codegen:v0.47.3-rc0-1-g7e27cad /repo
```

Comment:
How about specifying the image with sha256 hash, as specified in the PR's base commit?

Reply:
I see.
Is the sha256 hash better than specifying version?
I don't have a strong opinion, but I felt redundant hash with version comment like the base commit.

Reply:
The image specified with the tag can be overwritten, such as the git tag.
Conversely, the sha256 hash is immutable, such as the git commit hash.
So, specifying with the sha256 hash is more reproducible.

Reply:
I got your point! Thank you! I'll fix it.